markdownlint/docs/TOP010.md

```diff
@@ -1,10 +1,10 @@
 # `TOP010` - Use lazy numbering within ordered lists
 
-Tags: `ordered-list, lazy-numbering`
+Tags: `ol`
 
-Aliases: `use-lazy-numbering-in-ordered-lists`
+Aliases: `TOP010, useLazyNumbering`
```

Comment:
Only need the 2nd element from the rule's `names` array.
The suggestion uses the one from the suggestion from the second review comment.
```suggestion
Aliases: `lazy-numbering-for-ordered-lists`
```